h

```diff
@@ -0,0 +1,13 @@
+""" Helper tool for improved programming experience """
+
+import os
+import sys
+
+if sys.argv[1] == 'lint'.lower():
+    os.system('pylint ./**/*.py')
+
+elif sys.argv[1].lower() == 'test':
```

Comment:
Er det en fejl at .lower() er placeret ved 'lint' i linje 6, og ikke ved sys.argv[1], for at sikre keywordet "lint" ikke er case sensitive når du kører kommandoen fra terminalen?